src/risk_distributions/risk_distributions.py

```diff
@@ -52,6 +52,7 @@ def computable_parameter_index(mean: pd.Series, sd: pd.Series) -> pd.Index:
     @staticmethod
     def compute_min_max(mean: pd.Series, sd: pd.Series) -> pd.DataFrame:
         """Gets the upper and lower bounds of the distribution support."""
+        # noinspection PyTypeChecker
```

Comment:
what does this mean?

Reply:
This suppresses pycharm's type checker for the following statement.  Static analysis tools don't like it when you do arithmetic with series and floats due to some issue in the way that pandas thinks about types.